application-analytics-default/src/test/java/com/xwiki/analytics/internal/RowEvolutionJsonNormalizerTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.analytics.internal;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+import org.xwiki.test.junit5.mockito.ComponentTest;
+import org.xwiki.test.junit5.mockito.InjectMockComponents;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Unit test for {@link RowEvolutionJsonNormaliser}
+ *
+ * @version $Id$
+ */
+@ComponentTest
+public class RowEvolutionJsonNormalizerTest extends JsonNormalizerTest
+{
+    @InjectMockComponents
+    private RowEvolutionJsonNormaliser rowEvolutionJsonNormaliser;
+
+    @Test
+    public void normalizeDataWithoutFilters() throws IOException
```

Comment:
You don't need the `public`, since it's the default anyway.
You can update it for the MostViewedPagesTest as well